### DISCUSSION TITLE
e_dom_optimization experiment is removed but it returns false/null (not true)

### DISCUSSION THREAD

**Discussion by hogash** [elementor/elementor#24970] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I was relying on `e_dom_optimization` experiment for some conditions, but I see it's now removed in 3.19 , which is fine but it returns false by default (instead of true).

### Describe the solution you'd like.

Returns true by default to maintain legacy.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by FauzanEdris** [comment]
up
